### PR TITLE
fixed flake8 errors for all files except __init__.py in case we need the imports

### PR DESCRIPTION
**Pull request by christiegan**
Updated:
/tests/emg-import-test.py
/easy_music_generator/__version.py
/easy_music_generator/easy_music_generator.py
/easy_music_generator/pregenerator.py
/easy_music_generator/preprocessor/chord_distribution.py

Mostly space and line fixes. PEP8 requires lines to be less than 80 characters, so formatting is now stranger but conforms to PEP8. Unused variables commented out.

### CODE REVIEW

File: easy_music_generator/preprocessor/chord_distribution.py [hunk 1]
```diff
@@ -51,7 +63,9 @@ def get_chord_probabilities(dictionary):
             dictionary[key] /= total
         return dictionary
 
+
 class NoChordsFoundException(Exception):
-    def __init__(self, message="The provided MIDI or MusicXML files do not contain any chords."):
+    def __init__(self, message="The provided MIDI or MusicXML "
+                               "files do not contain any chords."):
```
Comment (andreiabds) [inline easy_music_generator/preprocessor/chord_distribution.py:69]: Does this string message work? I believe this was broken down to follow PEP-8 for the number of characters in one line. 
I would recommend doing the following: 

  def __init__(self, 
                       message="The provided MIDI or MusicXML files do not contain any chords."):

File: easy_music_generator/easy_music_generator.py [hunk 2]
```diff
@@ -11,45 +11,50 @@ def __init__(self):
...
         BUNDLE_PATH = "lakh2_polyphony_rnn.mag"
 
         OUTPUT_PATH = output_path
 
         # 16 steps in a bar
         num_steps = str(16*bars)
 
-        command = 'polyphony_rnn_generate --bundle_file=' + BUNDLE_PATH + ' --output_dir=' + OUTPUT_PATH + ' --num_outputs=1 --num_steps=' + num_steps + ' --primer_melody="' + primer_string + '" --primer_pitches="' + backing_string + '" --condition_on_primer=true --inject_primer_during_generation=false'
-        
+        command = 'polyphony_rnn_generate --bundle_file=' +\
```
Comment (andreiabds) [inline easy_music_generator/easy_music_generator.py:41]: command = 'polyphony_rnn_generate \
                       --bundle_file=" + BUNDLE_PATH + " \
                       --output_dir="+ OUTPUT_PATH + " \
                       --num_outputs=1 \
                       --num_steps="+ num_steps +" \
                       --primer_melody=" + primer_string + " \
                       --primer_pitches="+ backing_string + " \
                       --condition_on_primer=true \
                       --inject_primer_during_generation=false'

File: easy_music_generator/easy_music_generator.py [hunk 3]
```diff
@@ -11,45 +11,50 @@ def __init__(self):
...
-        primer_melody = preg.generate_primer_melody(self.note_matrix, bars) 
+        primer_melody = preg.generate_primer_melody(self.note_matrix, bars)
         primer_string = '['
         for i in range(len(primer_melody)):
             primer_string += str(primer_melody[i]) + ', '
         primer_string += str(primer_melody[len(primer_melody)-1])
         primer_string += ']'
 
-        backing_chord = preg.generate_backing_chords(self.chord_distribution, bars)
+        backing_chord = preg.generate_backing_chords(self.chord_distribution,
```
Comment (andreiabds) [inline easy_music_generator/easy_music_generator.py:24]: I would just ignore PEP8 here.

File: easy_music_generator/easy_music_generator.py [hunk 4]
```diff
@@ -11,45 +11,50 @@ def __init__(self):
         self.note_matrix = None
         self.chord_distribution = None
 
-
-    def generate(self, tempo=90, bars=4, output_path='/music_generator_output'):
+    def generate(self, tempo=90, bars=4,
```
Comment (andreiabds) [inline easy_music_generator/easy_music_generator.py:14]: ignore pep8, or create the variable outside. 

filepath = '/music_generator_output'
def generate(self, tempo=90, bars=4, output_path=filepath):

**Review by andreiabds — APPROVED** [review]
Nothing major that would block merging the code.

File: easy_music_generator/preprocessor/chord_distribution.py (hunk 1 above)
Comment (christiegan) [reply easy_music_generator/preprocessor/chord_distribution.py]: I think both the string cutting and removing init's underscores don't work here, so I'm ignoring PEP8 for this one

File: easy_music_generator/easy_music_generator.py (hunk 4 above)
Comment (christiegan) [reply easy_music_generator/easy_music_generator.py]: Okay! Good idea, made the change

**Pull request merged**